foundation/web/web.go

```diff
@@ -73,7 +73,7 @@ func (a *App) Handle(method string, group string, path string, handler Handler,
 	h := func(w http.ResponseWriter, r *http.Request) {
 		ctx := r.Context()
 
-		span := trace.SpanFromContext(ctx)
+		ctx, span := AddSpan(ctx, "foundation.app.handle")
```

Comment:
I understand that.

However, wthat this change allows the process to do is:
- From the existing span, generate a new span
- With the new span, monitor the Handle function itself
- The rest of the functionality, such as `ctx = context.WithValue(ctx, key, &v)`, there are no other changes.

So, what we achieve with this is not needing to have a separate operation to fetch the span from the existing context, because we generate a new span anyway a few lines below.